dashboard/app/models/programming_environment.rb

```diff
@@ -87,7 +87,7 @@ def remove_serialization
   end
 
   def studio_documentation_path
-    if DCDO.get('use-studio-code-docs', false) && ['applab', 'gamelab', 'spritelab', 'weblab'].include?(name)
+    if ['applab', 'gamelab', 'spritelab', 'weblab'].include?(name)
       "/docs/#{name}"
```

Comment:
this can actually be removed and we can just use `programming_environment_path` for everything. The name check is because the `docs/` link doesn't work for javalab no matter what the flag's value was